src/algorithms/reco/FarForwardLambdaReconstructionConfig.h

```diff
@@ -0,0 +1,21 @@
+// SPDX-License-Identifier: LGPL-3.0-or-later
+// Copyright (C) 2025 Sebouh Paul
+#pragma once
+#include <float.h>
+#include <DD4hep/Detector.h>
+
+
+namespace eicrecon {
+
+  struct FarForwardLambdaReconstructionConfig {
+    /** transformation from global coordinates to proton-frame coordinates*/
+    double rot_y=-0.025;
```

Comment:
I'd strongly suggest a more descriptive name here, especially considering that a parameter with the same name is used for a different rotation in the neutral reconstruction!
```suggestion
    double global_to_proton_rotation=-0.025;
```